infrastructure/terraform/environments/development-eu-west-2/spartan1/vars

```diff
@@ -4,6 +4,8 @@ aws_region = "eu-west-2"
 environment = "spartan1"
 deploy_to = "development"
 state_prefix = "env:/development"
+workspace_key_prefix = "test-data-generator-stack"
+state_file_name = "test-data-generator.tfstate"
 
 # Allows internal access to servers from these addresses.
 internal_cidrs = "172.16.0.0/16,172.19.0.0/16,172.24.0.0/16,10.172.16.0/24,10.172.21.0/24"
```

Comment:
Can we pull these values from the remote network state instead of having them in config please

Reply:
Yep, done.